website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -0,0 +1,172 @@
+
+Beyond SSO, Unleash supports identity providers that enforce [Multi-Factor Authentication (MFA)](/using-unleash/compliance/fedramp#identification-and-authentication) across all user access points. Every authentication event is logged with detailed metadata including timestamp, IP address, user agent, and authentication method used, providing comprehensive audit trails for security teams.
+
+Authentication mechanisms include:
+
+-   **FedRAMP**: Meets federal authentication standards through strong authentication controls.
+-   **SOC 2**: Provides the access control mechanisms required for the Security trust criterion.
+-   **ISO 27001**: Addresses access control objectives related to user authentication and authorization.
+
+To configure SSO in Unleash, navigate to **Admin > Single sign-on** in the Unleash Admin UI.
+
+![In Unleash's Single Sign-On page, there are four tabs to set up Open ID Connect, SAML 2.0, traditional passwords, and SCIM.](/img/use-case-user-mgmt-saml.png)
+
+## Configure role-based access control for enterprise security
+
+[Role-based access control (RBAC)](/reference/rbac) allows for fine-grained permission management, ensuring that team members can only access and modify feature flags relevant to their specific roles and responsibilities. Unleash's permission system enables precise control over user access and capabilities:
+
+-   **FedRAMP**: Implements the principle of least privilege and provides the detailed access controls required for federal systems.
+-   **SOC 2**: Addresses the Security trust criterion by ensuring appropriate access restrictions and user management.
+-   **ISO 27001**: Supports access control objectives by enabling organizations to implement and maintain detailed access management policies.
+
+To configure RBAC in the Unleash Admin UI, go to **Admin > Roles** to view, create, and manage user roles.
+
+![Manage all user roles in the Unleash Admin UI.](/img/use-case-user-mgmt-root-roles.png)
+
+For more fine-tuned access controls, create [custom root roles](/how-to/how-to-create-and-assign-custom-root-roles) and [custom project roles](/how-to/how-to-create-and-assign-custom-project-roles), where you can define the privileges and limitations beyond the [predefined roles](/reference/rbac#predefined-roles) we have built into Unleash.
+
+## Network security using Unleash
+
+Different parts of an organization have different security requirements. Unleash provides network security, CORS, API security for access control mechanisms, and the ability to create hidden projects that restrict the visibility of sensitive feature flag configurations.
```

Comment:
I could be wrong, but hidden projects are not implemented at the Network level so this isn't appropriate for this section.